Library/Formula/websockify.rb

```diff
@@ -0,0 +1,70 @@
+require "FileUtils"
+
+class Websockify < Formula
+  homepage "https://github.com/kanaka/websockify"
+  url "https://github.com/kanaka/websockify/archive/v0.6.0.tar.gz"
+  sha1 "f2bffebc9e4d94ad3a329638bee63e899eaad9c0"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  def install
+    system "make"
+
+    Dir.chdir("other") do
+      system "make"
+    end
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+    bin.install Dir["#{libexec}/bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+
+    # Move this so it is not installed with the other docs
+    move 'docs/websockify.1', 'websockify.1'
+    (share+"man"+"man1").install 'websockify.1'
+    (doc+"websockify").install 'docs'
+
+    # Upstream Bug: Not named '.dylib'
+    lib.install 'rebind.so'
+
+    # Install C version of websockify
+    move 'other/websockify', 'other/websockify-c'
+    bin.install 'other/websockify-c'
+
+    # Clean up before installing client
+    Dir.mkdir 'client'
+    move 'wsirc.html', 'client/wsirc.html'
+    move 'wstelnet.html', 'client/wstelnet.html'
+    rmdir 'include/web-socket-js-project'
+    move 'include', 'client/include'
+    (share+"websockify").install 'client'
+
+    # Clean up before installing node js
+    move 'other/js', 'node-js'
+    (share+"websockify").install 'node-js'
+
+    # Clean up before installing Ruby Websockify
+    Dir.mkdir 'ruby'
+    move 'other/websocket.rb', 'ruby/websocket.rb'
+    move 'other/websockify.rb', 'ruby/websockify.rb'
+    (share+"websockify").install 'ruby'
+
+    # Clean up before installing Clojure Websockify
+    Dir.mkdir 'clojure'
+    move 'other/project.clj', 'clojure/project.clj'
+    move 'other/websockify.clj', 'clojure/websockify.clj'
+    (share+"websockify").install 'clojure'
+
+    # Binaries not installed in bin (as bin.install ):-
+    # - rebind: coding of discovery of rebind.so needs re-writing
+    # - other/wswrap: dependency wswrapper.so seems to be missing
+    # - other/launch.sh: coding of path discovery of dependencies is just too weird to work
```

Comment:
Copied into share for now as rebind is used occasionally. However, it would need a complete re-write to actually be FHS friendly...